.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+language: python
+python:
+  - "2.7"
+  - "3.6"
+# command to install dependencies
+install:
+  - pip install -U setuptools
+  - python setup.py install easy_install[testing]
```

Comment:
Neither travis nor me got this to work. I think python setup.py testing is enough.

Reply:
Yes you are right, I think it is fixed now.